src/components/List.js

```diff
@@ -1,14 +1,25 @@
 import React from 'react';
+import { useHistory } from 'react-router-dom';
 import useFirestore from '../hooks/useFirestore';
 
 const List = ({ token }) => {
   const { docs } = useFirestore(token);
+  const history = useHistory();
+
   return (
     <div>
       <h1>List</h1>
+      {docs.length === 0 && <p>Your shopping list is currently empty.</p>}
+      {docs.length === 0 && (
+        <button onClick={() => history.push('/add-item')}>Add an Item</button>
+      )}
       <ul>{docs && docs.map((doc) => <li key={doc.id}>{doc.itemName}</li>)}</ul>
     </div>
   );
 };
 
 export default List;
+
```

Comment:
We decided not to add in any pop up message or modal as that seemed like more of a fun styling thing that we could do when it's time to style the app, if the rest of the team decides that would be a good idea. Thought - since we have `docs.length === 0` twice, can we refactor and simplify?